cli/cmd/root.go

```diff
@@ -314,13 +318,25 @@ func (options *proxyConfigOptions) validate() error {
 
 	if options.proxyCPURequest != "" {
 		if _, err := k8sResource.ParseQuantity(options.proxyCPURequest); err != nil {
-			return fmt.Errorf("Invalid cpu request '%s' for --proxy-cpu flag", options.proxyCPURequest)
+			return fmt.Errorf("Invalid cpu request '%s' for --proxy-cpu-request flag", options.proxyCPURequest)
 		}
 	}
 
 	if options.proxyMemoryRequest != "" {
 		if _, err := k8sResource.ParseQuantity(options.proxyMemoryRequest); err != nil {
-			return fmt.Errorf("Invalid memory request '%s' for --proxy-memory flag", options.proxyMemoryRequest)
+			return fmt.Errorf("Invalid memory request '%s' for --proxy-memory-request flag", options.proxyMemoryRequest)
+		}
+	}
+
+	if options.proxyCPULimit != "" {
+		if _, err := k8sResource.ParseQuantity(options.proxyCPULimit); err != nil {
+			return fmt.Errorf("Invalid cpu limit '%s' for --proxy-cpu-limit flag", options.proxyCPULimit)
+		}
+	}
+
+	if options.proxyMemoryLimit != "" {
+		if _, err := k8sResource.ParseQuantity(options.proxyMemoryLimit); err != nil {
+			return fmt.Errorf("Invalid memory limit '%s' for --proxy-memory-limit flag", options.proxyMemoryLimit)
 		}
 	}
```

Comment:
To be more bulletproof, it would be great if you could add a validation to check requests are not greater than limits, if both are set.

Reply:
Fixed with 35eb8fe2743ad6c46eb235f1b301715ee1b9ea11, let me know what you think